cmd/juju/model/exportbundle.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+package model
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/juju/api/bundle"
+	"github.com/juju/juju/cmd/modelcmd"
+)
+
+// NewexportbundleCommand returns a fully constructed exportbundle command.
+func NewExportBundleCommand() cmd.Command {
+	return modelcmd.Wrap(&exportBundleCommand{})
+}
+
+type exportBundleCommand struct {
+	modelcmd.ModelCommandBase
+	api ExportBundleModelAPI
```

Comment:
I'd rather we had a function to obtain an api, similar to https://github.com/juju/juju/blob/develop/cmd/juju/storage/poolcreate.go#L55